Traceback (most recent call last):
  File "/home/xiang/workspace/benchmark/rulf3/main.py", line 240, in <module>
    print('missing', crate, path)
ValueError: I/O operation on closed file.
